### PR TITLE
Issue15 fe

### PR DESCRIPTION
**Pull request by kassimbashir**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -13,6 +13,11 @@ import Calendar from './pages/Calendar'
 import About from './pages/About'
 import ContactUs from './pages/ContactUs'
 import EditEntity from './pages/EditEntity'
+import adminsignin from './admin-pages/Signin'
+import AddAdmin from './admin-pages/AddAdmin'
```
Comment (Karyum) [inline src/App.jsx:17]: do you need this in the `App.jsx` ? my guess is no

File: src/App.jsx [hunk 2]
```diff
@@ -13,6 +13,11 @@ import Calendar from './pages/Calendar'
...
+import volunteerSignin from './volunteer-pages/Signin'
```
Comment (Karyum) [inline src/App.jsx:18]: component name always capital letter

File: src/App.jsx [hunk 3]
```diff
@@ -41,6 +46,10 @@ function App() {
             <Route path="/calendar/:publisherId/:year/:month" component={Calendar} />
             <Route path="/about" component={About} />
             <Route path="/contact-us" component={ContactUs} />
+            <Route path="/admin-signin" component={adminsignin} />
+            <Route path="/admin-add" component={AddAdmin} />
```
Comment (Karyum) [inline src/App.jsx:50]: we might not need these also

File: src/volunteer-pages/addVolunteer/index.tsx [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+  const onClick = () => {
+    setError('')
+
+    if (!matchedPassword()) {
+      setError('passwords dont match, check and start again')
+      return
+    }
+    axios
+      .post(process.env.REACT_APP_API_URL + '/Volunteer/add', newVolunteer)
```
Comment (Karyum) [inline src/volunteer-pages/addVolunteer/index.tsx:39]: it's a little bit annoying to have  `'/Volunteer/add'` here and  ` '/volunteer/signin'` in the last file 😅, for consistency keep it all lower cased

**Review by Karyum — CHANGES_REQUESTED** [review]
Good job, later on we will change the design for something a bit better but for now this is fine.

I have a couple of changes though:

1) Connect the navbar buttons with the pages
2) Since this is the user side and not the admin one, then it needs to be translated to Arabic.

File: src/App.jsx (hunk 1 above)
Comment (kassimbashir) [reply src/App.jsx:17]: no, sorry this was for testing. I will make sure to delete things that I add for testing

File: src/App.jsx (hunk 3 above)
Comment (kassimbashir) [reply src/App.jsx:50]: Yes this was for testing, will delete

File: src/volunteer-pages/addVolunteer/index.tsx (hunk 4 above)
Comment (kassimbashir) [reply src/volunteer-pages/addVolunteer/index.tsx:39]: alrighty

**Review by Karyum — APPROVED** [review]

**Review by Karyum — APPROVED** [review]

**Pull request merged**